### DISCUSSION TITLE
where is it?

### DISCUSSION THREAD

**Discussion by JoaoDias12** [community/community#29723] [Repositories] [answered]
I was creating a new repository, I put the html, css, js and json files there, and I created two folders one images and one assets, extract the images to the images folder, I created another folder inside the assets called fonts and two more inside the fonts call, demo-files and fonts, extract all the files and my icon simply does not appear, I did the same always and it is not appearing.

https://github.com/JoaoDias12/Pietra-Tattos

https://joaodias12.github.io/Pietra-Tattos/

print of how it was supposed to be: https://prnt.sc/GwS5uMzn4QjH

**Comment by PavelLaptev** [comment, marked as answer]
@JoaoDias12 remove `/` in the beginning of the path, and it should work 🙂 

the correct line will be `<link rel="stylesheet" href="assets/fonts/style.css">`

this line:
https://github.com/JoaoDias12/Pietra-Tattos/blob/c40b18e437daae428d4df6467e7b1f1c52301568/index.html#L21

**Reply by JoaoDias12** [reply]
thanks man you is my hero

**Reply by PavelLaptev** [reply]
@JoaoDias12 no prob, glad it helped 😉 can you please mark the question as answered
